src/navigations/HomeBottomTabs.js

```diff
@@ -104,7 +104,7 @@ function HomeBottomTabs() {
 
   const menuIndicator = (nav, route) => {
     const isAnonChatMenu = route.name === 'AnonymousChannelList';
-    const activeColor = isAnonChatMenu ? COLORS.anon_primary : COLORS.blueOnboarding;
+    const activeColor = isAnonChatMenu ? COLORS.anon_primary : COLORS.signed_primary;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `COLORS.blueOnboarding` to `COLORS.signed_primary` seems to be a simple color adjustment. As long as `COLORS.signed_primary` is defined and the color fits the UI design, this change should be fine.
```diff
-    const activeColor = isAnonChatMenu ? COLORS.anon_primary : COLORS.blueOnboarding;
+    const activeColor = isAnonChatMenu ? COLORS.anon_primary : COLORS.signed_primary;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->